src/test/integration/runner.js

```diff
@@ -0,0 +1,29 @@
+require("mocha/mocha");
+const expect = require("expect.js");
+
+const { prefs } = require("../../utils/prefs")
+const prettyPrint = require("./tests/pretty-print")
+
+function ok() {
+  expect(true).to.be.truthy
```

Comment:
function `ok()` does not take any parameters.
`expect(true).to.be.truthy` probably always returns true.